doc/manual/advanced-topics/cores-vs-jobs.xml

```diff
@@ -0,0 +1,120 @@
+<chapter xmlns="http://docbook.org/ns/docbook"
+      xmlns:xlink="http://www.w3.org/1999/xlink"
+      xmlns:xi="http://www.w3.org/2001/XInclude"
+      version="5.0"
+      xml:id="chap-tuning-cores-and-jobs">
+
+<title>Tuning Cores and Jobs</title>
+
+<para>Nix has two relevant settings with regards to how your CPU cores
+will be utilized: <xref linkend="conf-cores" /> and
+<xref linkend="conf-max-jobs" />.</para>
+
+<variablelist>
+  <varlistentry>
+    <term><xref linkend="conf-max-jobs" /></term>
+    <listitem><para>
+      Dictates how many separate derivations will be built at the same
+      time. If you set this to zero, the local machine will attempt to
+      do no builds (forcing it to be built remotely.)
+    </para></listitem>
+  </varlistentry>
+  <varlistentry>
+    <term><xref linkend="conf-cores" /></term>
+    <listitem><para>
+      Dictates how many cores each derivation is allowed to use. If
+      you set this to zero, the builds will be given access to all the
+      cores of your system.
```

Comment:
"Dictates" -> "suggests" (and drop the second sentence). There is nothing that stops a derivation from using more cores. In practice, the "cores" option translates to GNU Make's `-j <cores>` flag.